ci/ci/github.py

```diff
@@ -294,7 +294,9 @@ def github_status_from_build_state(self):
                 'ci',
                 f'/batches/{self.batch.id}')
         else:
-            target_url = f'https://ci.hail.is/watched_branches/{self.target_branch.index}/pr/{self.number}'
+            target_url = deploy_config.external_url(
+                'ci',
+                f'/watched_branches/{self.target_branch.index}/pr/{self.number}')
```

Comment:
I forgot the deploy config when I was fixing the production CI bug. I added the fix here.